ADAL/tests/integration/AADAuthorityValidationIntegrationTests.m

```diff
@@ -203,15 +205,14 @@ - (void)testAcquireToken_whenAuthorityInvalid_shouldReturnError
 
 - (void)testValidateAuthority_whenHostUnreachable_shouldFail
 {
-    NSString* authority = @"https://login.windows.cn/MSOpenTechBV.onmicrosoft.com";
-
+    NSString* authority = @"https://login.windows.net/contoso.com";
     
     ADAuthorityValidation* authorityValidation = [[ADAuthorityValidation alloc] init];
     ADRequestParameters* requestParams = [ADRequestParameters new];
     requestParams.authority = authority;
     requestParams.correlationId = [NSUUID UUID];
     
-    NSURL* requestURL = [ADAuthorityValidationRequest urlForAuthorityValidation:authority trustedHost:s_kTrustedAuthority];
```

Comment:
Should we delete s_kTrustedAuthority? Seems it's not used anywhere.

Reply:
👍